keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -175,29 +190,29 @@ NetworkPolicy to their Namespaces which isolates the workloads.
 
 ClusterNetworkPolicy `Pass` rules allows an admin to delegate security posture for
 certain traffic to the Namespace owners. For example, allowing intra-tenant traffic
-can be delegated to tenant admins explicitly by the cluster admin with the use of `pass`
-rules.
+can be delegated to tenant admins explicitly by the cluster admin with the use of
+`Pass` rules.
 
 #### Priority
 
 The policy instances will be ordered based on the numeric priority assigned to the
-CNP. A lower number corresponds to a higher order. For example, CNP with `priority` set
-to "0" will be considered as the highest priority policy, while a large number, i.e.
-999, will be considered as low priority. Any numeric-valued priority, however, will have
+CNP. A smaller number corresponds to a higher precedence. For example, CNP with `priority`
+set to "0" will be considered as the policy with the highest precedence, while "999" will
+be considered as low precedence. Any numeric-valued priority, however, will have
 higher precedence over the namespaced NetworkPolicy instances in the cluster.
 
 Additionally, a special keyword `Baseline` can be used in the priority field to indicate
-that the rules in that policy instance shall be created at a priority lower than the Namespaced
-NetworkPolicies. Admission controllers will need to validate that only this string value or
-numeric values are set for the priority field.
+that the rules in that policy instance shall be created at a priority lower than the
+Namespaced NetworkPolicies. API schema validation will ensure that only numneric
+value or `Baseline` will be set for the priority field.
 
 (TODO: double check this. There are benefits of having 0 be the lowest priority so that there is
```

Comment:
I would remove this TODO at this point.... I feel like we've had enough feedback from the community